Additions/UIView-KIFAdditions.m

```diff
@@ -225,9 +225,26 @@ - (UIAccessibilityElement *)accessibilityElementMatchingBlock:(BOOL(^)(UIAccessi
     }
     
     if (!matchingButOccludedElement && self.window) {
+
+        CGPoint scrollContentOffset = {-1.0, -1.0};
+        UIScrollView *scrollView = nil;
         if ([self isKindOfClass:[UITableView class]]) {
+            //special case for UIPickerView (which has a private class UIPickerTableView)
+
+            for (UIView *view in [self subviews])
+            {
+                NSString * subViewName = [NSString stringWithFormat:@"%@", [view class]];
+                if ([subViewName containsString:@"UIPicker"] )
```

Comment:
Should we consider doing this for all table views and not just ones for picker views? (e.g. leave the views scrolled to their original position if they aren't a match). I saw some cases where the UI was scrolled to a position that is different than before.

This isn't necessary, but it would be nice to have consistency with the previous behavior and that of a user who isn't interacting with the UI leaving it unmodified.

Reply:
I decided to leave this alone. There was a reason we chose to do it this way, and I can't recall the exact thought behind it. We have some mileage on this since last Nov and I don't want to introduce something that breaks the fix.